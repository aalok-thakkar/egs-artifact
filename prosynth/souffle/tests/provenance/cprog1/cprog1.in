setdepth 8
explain loop(1, 4, 10, 100)
explain loop(2, 10, 10, 100)
exit
