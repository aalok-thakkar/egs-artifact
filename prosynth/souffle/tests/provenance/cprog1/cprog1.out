loop(0, 0, 10, 100)                                                                                                                 
----------------(R1)                                                                                                                
loop(1, 0, 10, 100)  loop(0, 0, 10, 100) 0 < 10                                                                                     
--------------------------------------------(R2)                                                                                    
              loop(1, 1, 10, 100)                loop(0, 0, 10, 100) 1 < 10                                                         
------------------------------------------------------------------------(R2)                                                        
                            loop(1, 2, 10, 100)                              loop(0, 0, 10, 100) 2 < 10                             
----------------------------------------------------------------------------------------------------(R2)                            
                                          loop(1, 3, 10, 100)                                            loop(0, 0, 10, 100) 3 < 10 
--------------------------------------------------------------------------------------------------------------------------------(R2)
                                                        loop(1, 4, 10, 100)                                                         
subproof loop() loop(0, 0, 10, 100) 4 < 10                                                                                                                                                       
----------------------------------------(R2)                                                                                                                                                      
            loop(1, 5, 10, 100)              loop(0, 0, 10, 100) 5 < 10                                                                                                                           
--------------------------------------------------------------------(R2)                                                                                                                          
                          loop(1, 6, 10, 100)                            loop(0, 0, 10, 100) 6 < 10                                                                                               
------------------------------------------------------------------------------------------------(R2)                                                                                              
                                        loop(1, 7, 10, 100)                                          loop(0, 0, 10, 100) 7 < 10                                                                   
----------------------------------------------------------------------------------------------------------------------------(R2)                                                                  
                                                      loop(1, 8, 10, 100)                                                        loop(0, 0, 10, 100) 8 < 10                                       
--------------------------------------------------------------------------------------------------------------------------------------------------------(R2)                                      
                                                                    loop(1, 9, 10, 100)                                                                      loop(0, 0, 10, 100) 9 < 10           
------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------(R2)          
                                                                                  loop(1, 10, 10, 100)                                                                                   10 >= 10 
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------(R3)
                                                                                       loop(2, 10, 10, 100)                                                                                       
